### PR TITLE
RWD added and links

### PR DESCRIPTION
**Pull request by mscibisz**
links were added to source code and RWD (also for mobile phones)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -200,4 +215,28 @@ sup {color: #585858;}
 
 sup {
     font-size: 9px;
+}
+
+
+
+
```
Comment (pietrushnic) [inline css/style.css]: @mscibisz do we need those redundant lines?

File: css/style.css [hunk 2]
```diff
@@ -200,4 +215,28 @@ sup {color: #585858;}
...
+}
+.source-code-column {
+    max-width: 100%;
+    display: block;
+}
+
+
+
+
+
```
Comment (pietrushnic) [inline css/style.css]: @mscibisz also here

**Comment by mscibisz** [comment]
Spaces and breaks were removed where unnecessary. The css file is ok now.
Any comments about the content? Anything to add or improve?

**Comment by pietrushnic** [comment]
@mscibisz  no additional comments, we wait for users review.

**Pull request merged**